week-5/B-callbacks/exercise.js

```diff
@@ -0,0 +1,83 @@
+/*
+  Callbacks
+  ---------------------------------
+  Open index.html in your browser
+*/
+
+/*
+  EXERCISE 1
+  =======
+
+  Write a function that inserts your name into the #exercise1 element when
+  the button is clicked
+*/
+//var name = "hasanein";
+function callEvent(name){
```

Comment:
I don't think this function is necessary. You can just set up the event listener directly